File: 2nd sem/assignment 4/p1.c
#include<stdio.h>
int sum(int *a,int *b);
void main()
{
	int a,b,s;
	printf("Enter 2 number: ");
	scanf("%d%d",&a,&b);
	s=sum(&a,&b);
	printf("sum of %d and %d is %d",a,b,s);
}
int sum(int *a,int *b)
{
	return (*a+*b);
}

File: 3rd sem/IT Lab/assignment 14(test)/p1.py
'''1) Write a program which repeatedly reads numbers until the user enters “done”. Once
“done” is entered, print out the total, count, and average of the numbers. If the user
enters anything other than a number, detect their mistake using try and except and print
an error message and skip to the next number.'''
print('Enter some numbers(Done to exit) ')
iList=[]
while(True):
    a=input('->>')
    if(a.lower()=='done'):
        break
    try:
        i=float(a)
        iList.append(i)
    except:
        print('Error: only numbers allowed')
c=len(iList)
t=sum(iList)
av=t/c
print(f'Total: {t}\nCount:{c}\nAverage: {av}')
